src/middleware.js

```diff
@@ -1,12 +1,32 @@
 import * as defaultOptions from './defaults';
 import { getActionTypes } from './getActionTypes';
+import axios from 'axios';
+
+let interceptorsBound = false;
+
+function bindInterceptors(getState, { requestInterceptors = [], responseInterceptors = [] } = {}) {
```

Comment:
Let's keep axios api please, so this should be `interceptors.request` and `interceptors.response`, please rename the keys.